sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -647,17 +648,17 @@
             "location": "[parameters('location')]"
         },
         {
-            "condition": "[and(variables('enablePE'), equals(parameters('storageAccountOption'), 'new'))]",
+            "condition": "[and(variables('enablePE'), or(equals(parameters('storageAccountOption'), 'new'), and(equals(parameters('set_up_feature_store'), 'true'), empty(parameters('offline_store_target')))))]",
```

Comment:
do we need the parameter of "set_up_feature_store"? can we use "kind" parameter for this?

Reply:
we can use either of them.